orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/api/manifests/BakeManifestRequest.java

```diff
@@ -18,24 +18,16 @@
 package com.netflix.spinnaker.orca.bakery.api.manifests;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.netflix.spinnaker.kork.artifacts.model.Artifact;
-import java.util.List;
-import java.util.Map;
 import lombok.Data;
 
 @Data
 public class BakeManifestRequest {
   @JsonProperty("templateRenderer")
   String templateRenderer;
 
-  @JsonProperty("inputArtifacts")
-  List<Artifact> inputArtifacts;
-
-  List<Artifact> values;
-
   @JsonProperty("outputName")
   String outputName;
 
-  @JsonProperty("overrides")
-  Map<String, Object> overrides;
+  @JsonProperty("outputArtifactName")
+  String outputArtifactName;
```

Comment:
I realize this is pre-existing, but given the scope of changes to this class (and the subclasses) would be good to make these member variables `private` as there's no need for them to be `public` given that `@Data` will generate an accessor.

(In an ideal world it would be great for them to be immutable as well and make everything `private final` but that might require some more changes to work with deserialization.)